plugins/databases/snowflake/src/main/java/org/apache/hop/core/database/SnowflakeDatabaseMeta.java

```diff
@@ -0,0 +1,401 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2017 by Hitachi Vantara : http://www.pentaho.com
+ *
```

Comment:
remove the pentaho data integration and vantara copyright, they will be mentioned in the notice file when we create a release, all file headers will be reviewed